sippy-ng/src/component_readiness/CompReadyUtils.js

```diff
@@ -214,9 +218,9 @@ export function getStatusAndIcon(status, grayFactor = 0) {
 // The values of a column's key/value pairs (except status) are
 // concatenated to form a column name
 export function formColumnName(column) {
-  return Object.keys(column)
-    .filter((key) => key != 'status' && key != 'regressed_tests')
-    .map((key) => column[key])
+  let variants = column['variants']
+  return Object.keys(variants)
+    .map((key) => variants[key])
     .join(' ')
 }
```

Comment:
There's a usage of this in the regressed tests modal that doesn't match the new style.  That modal looks broken actually

https://github.com/openshift/sippy/blob/960454694ecb62ef79ac1a3f1d807ed81f528152/sippy-ng/src/component_readiness/RegressedTestsModal.js#L37